frontend/src/pages/InvoicesPage/DownloadZipButton.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
```

Comment:
Can we extract the divergence with DownloadButton as parameter and avoid duplicating the class?